content/en/docs/400_growing-calendar/401_how-to/index.md

```diff
@@ -1,27 +1,26 @@
 ---
-title: "How can I use the Growing Calendar?"
+title: "How do I use the Growing Calendar?"
 description: "Learn how to use the Growing Calendar to plan your plantings"
 weight: 401
 draft: false
 url: "growing-calendar/how-to-use"
 ---
 
-Use the **Growing Calendar** in Planter to time your garden plantings!
+Use the **Growing Calendar** in Planter to see which plants you can start inside, transplant, or sow outside at any point in your local growing season.
 
-Check which plants you can **Start Inside**, **Transplant**, or **Sow Outside** at any point in your local growing season.
-
-{{< alert icon="🌶️" context="warning" text="Be sure to [**set your frost dates**](../../getting-started/frost-dates) so the Growing Calendar will show the correct timings for your area" />}}
+{{< alert icon="🌶️" context="warning" text="Be sure to [**set your frost dates**](../../getting-started/frost-dates) so the Growing Calendar will show accurate information for your area" />}}
 {{< alert icon="🌴" context="info" text="If you don't have frost or freezing temperatures in your area, ignore the **Growing Calendar** dates. Plant cool season crops in your cooler season and plant heat-loving crops in your warmer season." />}}
```

Comment:
Please add a link to our blog post which explains more: https://blog.planter.garden/posts/growing-veggies-in-subtropical-and-tropical-climates/